src/services/resource-creator.js

```diff
@@ -5,10 +5,14 @@ const utils = require('../utils/schema');
 
 function ResourceCreator(Model, params) {
   const schema = Interface.Schemas.schemas[utils.getModelName(Model)];
-
   function create() {
     return new P((resolve, reject) => {
-      if ('_id' in params) { delete params._id; }
+      const idField = schema.fields.find((field) => field.field === '_id');
+      const isAutomaticId = !idField || idField.isGenerated;
+
+      if ('_id' in params && isAutomaticId) {
```

Comment:
Same question here to be sure to understand:
Can we have a "readonly" _id field that is not "generated"?

Reply:
@arnaudbesnier Yes: https://github.com/ForestAdmin/forest-express-mongoose/pull/393/files/ef465d14ea0951b9602ef9da527f70ddde5dff08#r479938255